api/can_api_v2_definition.py

```diff
@@ -346,6 +359,12 @@ class Metrics(base_model.APIBaseModel):
     vaccinationsCompletedRatio: Optional[float] = pydantic.Field(
         None, description=("Ratio of population that has completed vaccination.")
     )
+    vaccinationsAdditionalDoseRatio: Optional[float] = pydantic.Field(
+        None,
+        description=(
+            "Ratio of population that are fully vaccinated and received a booster (additional) dose."
```

Comment:
nits:
* "received" => "have received" (to match dose count description above)
* "additional" => "or additional" (to match dose count description above)